samples/graph-bulk-meetings/csharp/EventMeeting/ClientApp/src/components/FileUpload.jsx

```diff
@@ -0,0 +1,154 @@
+﻿import React, { Component } from 'react';
+import "../style/style.css";
+import { format } from 'date-fns';
+import { OutTable, ExcelRenderer } from 'react-excel-renderer';
+import { Col, Input, InputGroup, FormGroup, Label, Button, Fade, FormFeedback, Container, Card } from 'reactstrap';
+import * as microsoftTeams from "@microsoft/teams-js";
+import axios from "axios";
+microsoftTeams.initialize();
+
+class App extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            isOpen: false,
+            dataLoaded: false,
+            isFormInvalid: false,
+            header: 0,
+            blankrows: false,
+            rows: null,
+            cols: null
+        }
+        this.fileHandler = this.fileHandler.bind(this);
+        this.toggle = this.toggle.bind(this);
+        this.openFileBrowser = this.openFileBrowser.bind(this);
+        this.renderFile = this.renderFile.bind(this);
+        this.openNewPage = this.openNewPage.bind(this);
+        this.fileInput = React.createRef();
+    }
+    renderFile = (fileObj) => {
+        //just pass the fileObj as parameter
+        ExcelRenderer(fileObj, (err, resp) => {
+            if (err) {
+                console.log(err);
+            }
+            else {
+                this.setState({
+                    dataLoaded: true,
+                    cols: resp.cols,
+                    rows: resp.rows
+                });
+                console.log(resp.cols, resp.rows);
+            }
+        });
+    }
+
+    fileHandler = (event) => {
+        if (event.target.files.length) {
+            let fileObj = event.target.files[0];
+            let fileName = fileObj.name;
+
+
+            //check for file extension and pass only if it is .xlsx and display error message otherwise
+            if (fileName.slice(fileName.lastIndexOf('.') + 1) === "xlsx") {
+                this.setState({
+                    uploadedFileName: fileName,
+                    isFormInvalid: false
+                });
+                this.renderFile(fileObj)
+            }
+            else {
+                this.setState({
+                    isFormInvalid: true,
+                    uploadedFileName: ""
+                })
+            }
+        }
+    }
+
+    // Handler when user click on create button.
+    onCreateTeamsMeetingsButtonClick = (event) => {
```

Comment:
Change to onCreateTeamsMeetingButtonClick

Reply:
Resolved